src/com/facebook/buck/swift/SwiftDescriptions.java

```diff
@@ -56,6 +56,10 @@ public static boolean isSwiftSource(
     return swiftSrcsBuilder.build();
   }
 
+  public static boolean hasSwiftSource(SourcePathResolverAdapter sourcePathResolverAdapter, ImmutableSet<SourceWithFlags> srcs) {
+    return filterSwiftSources(sourcePathResolverAdapter, srcs).size() > 0;
```

Comment:
At least in Swift code we tend to prefer `!collection.isEmpty` to `collection.count > 0` because `isEmpty` is a faster calculation than grabbing `count`. Not sure if that applies here.

Reply:
That makes sense. Changed. Thanks!